AppServer/google/appengine/api/logservice/logservice_stub.py

```diff
@@ -31,6 +31,8 @@
 from google.appengine.api.logservice import log_service_pb
 from google.appengine.runtime import apiproxy_errors
 
+sys.path.append(os.path.join(os.path.dirname(__file__), "../../../../../lib"))
```

Comment:
I understand this change, but why do you think it is better to replace the symlink with adding the LogService directory in the sys path?
Its just that there are other python files that get loaded in the python vm's sys path that we do not need here.
Actually both ways are not ideal.